src/BitPaySDK/Exceptions/WalletQueryException.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace BitPaySDK\Exceptions;
+
+
+class WalletQueryException extends RefundException
```

Comment:
make sure this follows the same structure across exceptions, WalletQueryException should extend a generic WalletException